source/client/cl_presence.c

```diff
@@ -596,8 +596,7 @@ static const char *CL_PlayerStatus( snapshot_t *frame )
 
 void CL_UpdatePresence( void )
 {
-	// is there a better way of checking whether steam is ready?
-	if( cl_presence_state.discord_initialized || Steam_Active()) {
```

Comment:
what did you find?

Reply:
The problem is the "Main Menu" "Ready to Fork" discord status doesn't appear unless you connect and disconnect from a server. After he removed removing Steam_Active() if Discord isn't running then Steam will no longer have status updates.

Reply:
> what did you find?

the comment was made before i had rewritten steamshim, it had different behavior then